website/pages/blog/product-updates-3.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "CloudQuery Product Updates #3"
+tag: product
+date: 2023/01/23
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here! This month was packed!
+
+## SDK Updates
+
+- We now support [incremental syncing](https://www.cloudquery.io/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.
+
+## Destinations
+
+### New Destinations
+
+We've added 7 new destinations to CQ!!
+
+* [MongoDB](https://www.cloudquery.io/docs/plugins/destinations/mongodb/overview)
+* [Neo4j](https://www.cloudquery.io/docs/plugins/destinations/neo4j/overview)
+* [Kafka](https://www.cloudquery.io/docs/plugins/destinations/kafka/overview)
+* [MSSQL](https://www.cloudquery.io/docs/plugins/destinations/mssql/overview)
+* [GCS](https://www.cloudquery.io/docs/plugins/destinations/gcs/overview)
+* [S3](https://www.cloudquery.io/docs/plugins/destinations/s3/overview)
+* [File](https://www.cloudquery.io/docs/plugins/destinations/file/overview)
+* [Azure Blob Storage](https://www.cloudquery.io/docs/plugins/destinations/azblob/overview)
+
+## Sources
+
+### New Sources
+
+We've added 7 new source plugins to CQ to cover the long tail of infrastructure apps!
+
+* [Slack](https://www.cloudquery.io/docs/plugins/sources/slack/overview)
+* [PagerDuty](https://www.cloudquery.io/docs/plugins/sources/pagerduty/overview)
+* [Alibaba Cloud](https://www.cloudquery.io/docs/plugins/sources/alicloud/overview)
+* [Oracle](https://www.cloudquery.io/docs/plugins/sources/oracle/overview)
+* [GitLab](https://www.cloudquery.io/docs/plugins/sources/gitlab/overview)
+* [HackNews](https://www.cloudquery.io/docs/plugins/sources/hackernews/overview)
+* [Shopify](https://www.cloudquery.io/docs/plugins/sources/shopify/overview)
+* [Stripe](https://www.cloudquery.io/docs/plugins/sources/stripe/overview)
+
+Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/CHANGELOG.md).
```

Comment:
```suggestion
Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/40ff6b263b4ad0e96b00495731ed8f4247f87b52/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/40ff6b263b4ad0e96b00495731ed8f4247f87b52/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/40ff6b263b4ad0e96b00495731ed8f4247f87b52/plugins/source/azure/CHANGELOG.md).
```

We should pin these files to a specific commit so their content reflects the time the blog is published (you can do this by clicking `y` on your keyboard when you have a file open)